### PR TITLE
action: add feedback_comment_creator to improve filtering

### PR DESCRIPTION
**Pull request by goapunk**
analogue to #1412

### CODE REVIEW

**Review by fuzzylogic2000 — COMMENTED** [review]
It does look good to me. Why is it WIP?

File: adhocracy4/actions/signals.py [hunk 1]
```diff
@@ -30,21 +30,27 @@ def _add_action(sender, instance, created, update_fields, **kwargs):
...
         target = None
         target_creator = None
+        feedback_comment_creator = None
         if created:
             target = _extract_target(instance)
             if target:
                 verb = Verbs.ADD.value
                 if hasattr(target, "creator"):
                     target_creator = target.creator
+                    print(target_creator)
```
Comment (Rineee) [inline adhocracy4/actions/signals.py:40]: uups :)

**Comment by goapunk** [comment]
good point, `obj_comment_creator` is better, will change it.

**Review by Rineee — APPROVED** [review]
Cool!

**Pull request merged**